lib/Sema/CSDiag.cpp

```diff
@@ -735,7 +735,7 @@ void FailureDiagnosis::diagnoseUnviableLookupResults(
 
   ValueDecl *member = nullptr;
   for (auto cand : result.UnviableCandidates) {
-    if (member == nullptr)
+    if (member == nullptr && cand.isDecl())
```

Comment:
Looks like this is actually dead code, you can just delete the whole block.

Reply:
Fantastic! I've done so. Tx.